code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -27,9 +27,10 @@
 	..()
 
 
-/obj/item/robot_module/New()
+/obj/item/robot_module/New(loc, should_have_flash = TRUE)

```

Comment:
Sidenote: A lot of these procs missing args will be fixed when I add in dreamchecker, which should kill off a fair few runtimes.

Reply:
Dreamchecker will be nice